.github/workflows/ci-sage.yml

```diff
@@ -0,0 +1,173 @@
+name: Run Sage CI for Linux/Cygwin/macOS
```

Comment:
I think this is just Linux.

Reply:
True, I cut out the other platforms to make the file simpler. Given that sympy is pure Python, there's probably relatively little point in the full portability testing that we do.